docs/source/configuration/multilingual.md

```diff
@@ -16,7 +16,13 @@ You can enable it through the {guilabel}`Add-ons` control panel in the administr
 
 ## Volto configuration
 
-You need to enable multilingual in Volto's configuration object.
+### In Volto 19 and later
+
+Volto 19+ will automatically load the current settings for default language and available languages from the Plone site. A site admin can edit these settings in the Languages control panel in Site Setup, or a developer can configure them using GenericSetup.
```

Comment:
````suggestion
In Volto 19 and later, both the default language and available languages are defined in the Plone site configuration.
A site administrator can configure these values in the {guilabel}`Languages` control panel in {guilabel}`Site Setup`.
A developer can configure them using GenericSetup.

```{versionchanged} Volto 19
Use multilingual configuration from the backend, instead of an environment variable for the frontend.
```
````